rome-modules/src/test/java/com/rometools/modules/mediarss/MediaModuleTest.java

```diff
@@ -91,6 +89,30 @@ public void testParse() throws Exception {
         }
     }
 
+
+    public void testParseFileSizeWithUnits() throws Exception {
+        final SyndFeed feed = getSyndFeed("org/rometools/feed/module/mediarss/filesize-with-unit.xml");
+        final List<SyndEntry> entries = feed.getEntries();
+
+        final MediaEntryModule m1 = (MediaEntryModule) entries.get(0).getModule(MediaModule.URI);
+        assertThat(m1.getMediaContents()[0].getFileSize(), is(valueOf(2).multiply(valueOf(1024).pow(3)).longValue()));
+
+        final MediaEntryModule m2 = (MediaEntryModule) entries.get(1).getModule(MediaModule.URI);
+        assertThat(m2.getMediaContents()[0].getFileSize(), is(valueOf(1.1).multiply(valueOf(1024).pow(3)).longValue()));
```

Comment:
It's not obvious what exactly you are testing here. Maybe have a separate test for each assert and use method name to describe what the test is about?